### PR TITLE
13.0 purchase incoming products

### PR DESCRIPTION
**Pull request by Tino480**
This module adds a feature from 12.0 that was removed;
It adds a menu item to generate a view showing all products to receive.

### CODE REVIEW

**Review by alan196 — CHANGES_REQUESTED** [review]
@Tino480 Please make only one commit for this change.

Also give a description to this PR

👍

**Review by alan196 — APPROVED** [review]

File: purchase_incoming_products/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020, Jarsa Sistemas, S.A. de C.V.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Incoming Products In Purchase",
+    "summary": "Module that generates a view for incoming products",
+    "version": "13.0.1.0.0",
+    "category": "report",
+    "author": "Jarsa Sistemas, S.A. de C.V.," "Odoo Community Association (OCA)",
```
Comment (HviorForgeFlow) [inline purchase_incoming_products/__manifest__.py]: Drop spare quotation marks

File: purchase_incoming_products/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2020, Jarsa Sistemas, S.A. de C.V.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    "name": "Incoming Products In Purchase",
+    "summary": "Module that generates a view for incoming products",
+    "version": "13.0.1.0.0",
+    "category": "report",
```
Comment (HviorForgeFlow) [inline purchase_incoming_products/__manifest__.py]:
```suggestion
    "category": "Reporting",
```

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Hello @Tino480, functional review :+1:

Code review comments:
- Could you change commit description to something like: [13.0][ADD] purchase_incoming_products
- As you are taking code directly from Odoo v12 you should add Odoo S.A. as an author also.

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**